pkg/utils/file.go

```diff
@@ -81,15 +92,20 @@ func Exists(name string) bool {
 	return true
 }
 
+// CreateNestedDirectory create nested directory
+func CreateNestedDirectory(path string) error {
+	err := os.MkdirAll(path, 0700)
```

Comment:
no need to check for existence for os.MkdirAll, won't fail if path already exists